docs/manuals/gui/viselements/charts/funnel.md

```diff
@@ -0,0 +1,359 @@
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:chart type="funnel" x="Visits" y="Opps">{data}</taipy:chart>
+        ```
+
+The resulting chart looks like this:
+
+<figure>
+    <img src="../funnel-simple-d.png" class="visible-dark" />
+    <img src="../funnel-simple-l.png" class="visible-light"/>
+    <figcaption>Simple funnel chart</figcaption>
+</figure>
+
+
+### Multiple traces {data-source="gui:doc/examples/charts/funnel-multiple"}
+
+With the previous example, we could spot how many contacts would convert to sales,
+globally for a company.<br/>
+We can split this information into Regions and get more detailed conversion
+information.
+
+In the following code, we have the same information spread across regions:
+
+```py
+data = {
+    # The stage names
+    "Types": ["Website visit", "Downloads", "Prospects", "Invoice sent", "Closed"],
+    # Volume for each stage, in the US
+    "Visits_us": [13873, 10533, 5443, 2703, 908],
+    # Volume for each stage, in the EU
+    "Visits_eu": [ 7063, 4533, 3443, 1003, 1208 ],
+    # Volume for each stage, in the AP region
+    "Visits_ap": [ 6873, 2533, 3443, 1703, 508 ]
+}
+
+# Columns for each trace
+x = ["Visits_us", "Visits_eu", "Visits_ap"]
+
+# Legend text for each trace
+names = ["US", "EU", "AP"]
+```
+
+*x* is an array of the entries in the data set that we use as the
+value of the [*x*](../chart.md#p-x) property of the chart control.<br/>
+It is a shortcut to setting each [*x[]*](../chart.md#p-x) value, where we
+would have had to write, in the chart definition, the pretty long:
+`x[1]=Visits_us|x[2]=Visits_eu|x[3]=Visits_ap`, and even longer in HTML.
+
+The *names* variable holds all the trace names, so they appear properly in the chart
+legend.
+
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{data}|type=funnel|chart|x={x}|y=Types|name={names}|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:chart type="funnel" x="{x}" y="Types" name="{names}">{data}</taipy:chart>
+        ```
+The resulting chart looks like this:
+
+<figure>
+    <img src="../funnel-multiple-d.png" class="visible-dark" />
+    <img src="../funnel-multiple-l.png" class="visible-light"/>
+    <figcaption>Funnel chart with multiple traces</figcaption>
+</figure>
+
+### Styling {data-source="gui:doc/examples/charts/funnel-styling"}
+
+Taipy lets you customize how the funnel chart elements can be styled.<br/>
+To indicate how bars and lines should look, use the
+[*marker*](../chart.md#p-marker) and [*options*](../chart.md#p-options) properties
+of the chart control.
+
+Here is some code that shows how styling can be customized:
+
+```py
+data = {
+    "Types":  ["Website visit", "Downloads", "Prospects", "Invoice sent", "Closed"],
+    "Visits": [13873, 10533, 5443, 2703, 908]
+}
+
+marker = {
+    # Boxes are filled with a blue gradient color
+    "color": ["hsl(210,50%,50%)", "hsl(210,60%,60%)", "hsl(210,70%,70%)", "hsl(210,80%,80%)", "hsl(210,90%,90%)"],
```

Comment:
color could also point the a column in the data set

Reply:
Yes. That would be a more general indication, but yes.

Reply:
Done in the 'key properties section'.